contents/docs/experiments/manual.mdx

```diff
@@ -8,136 +8,314 @@ availability:
+## How to create an experiment in PostHog
 
-After clicking on "new experiment" in the Experiments page, we should see a form like this:
+To create a new experiment, go to the [experiments tab](https://app.posthog.com/experiments) in the PostHog app, and click on the "New experiment" button in the top right.
 
-![Create experiment](../../images/features/experiments/create-experiment.png)
+This presents you with a form where you can complete the details of your new experiment:
 
-#### 1. Experiment name, feature flag, and testing variants
+![How to create an experiment in PostHog](../../images/features/experiments/create-experiment.png)
 
-Let's start in the upper left corner by filling out a name, description, and feature flag for our experiment. Fill in a name for your feature flag - it will be created here by default, linked to the experiment rather than existing as its own entity.
+Here's a breakdown of each field in the form:
 
-![Name, feature flag, and variants](../../images/features/experiments/create-experiment-name-variants.png)
+### Feature flag key
 
-We can also include multivariants for our testing changes. For example, instead of running our experiment with just "control" and "test", we can test for the green visual button change or the orange one or our original control.
+Each experiment is backed by a [feature flag](/docs/feature-flags/manual). In your code, you use this feature flag key to check which experiment variant the user has been assigned to. 
 
-#### 2. Participant selection and experiment goal/metric
+> **Advanced:** It's possible to create experiments without using PostHog's feature flags (for example, if you you're a different library for feature flags). For more info, read our docs on [implementing experiments without feature flags](// TODO add link here) 
 
-Next, we want to select our experiment's universe. Here, we've decided that for our sign up button experiment we want to test our hypothesis on users from Asia.
+### Experiment variants
 
-We also want to set our experiment's "goal" now for which metric we want to track. Since we're testing whether the number of sign up button clicks increase or not, we'll use the "Trend" goal to measure aggregate values
+By default, all experiments have a 'control' and 'test' variant. If you have more variants that you'd like to test, you can use this field to add more variants and run multivariant tests. Participants are automatically split equally between variants.
 
-![Participants and goal metric](../../images/features/experiments/create-experiment-participants-goals.png)
+### Participant type
 
-If we want to test a hypothesis for a conversion rate metric, we'll use the "Funnel" goal instead.
+The default is users, but if you've created [groups](/docs/product-analytics/group-analytics) in PostHog, you can use this field to run [group-targeted experiments](/blog/running-group-targeted-ab-tests). This will test how a change affects your product at a group-level. 
```

Comment:
```suggestion
The default is users, but if you've created [groups](/docs/product-analytics/group-analytics), you can run [group-targeted experiments](/blog/running-group-targeted-ab-tests). This will test how a change affects your product at a group-level by providing the same variant to every member of a group.
```